README.md

```diff
@@ -32,6 +32,7 @@
 `@octokit/webhooks` helps to handle webhook events received from GitHub.
 
 [GitHub webhooks](https://docs.github.com/webhooks/) can be registered in multiple ways
+![image](https://user-images.githubusercontent.com/4595477/211971688-c0148b26-a4aa-4f24-b8ef-aaddd6368cfc.png)
```

Comment:
```suggestion
⚠️ 
```
![Screenshot 2023-01-12 at 10 06 28 AM](https://user-images.githubusercontent.com/139819/212119153-c1f1e55b-ba18-405b-9cb6-112a91b87e41.png)

This is rendering a little strange - perhaps we could just use the ⚠️ emote.

Reply:
That should be the emoji 😅

When you copy and paste strings using emojis from GitHub, it includes an image instead of actual emoji characters